go/stats/export_test.go

```diff
@@ -129,3 +131,7 @@ func TestDropVariable(t *testing.T) {
 	_ = NewGaugesWithSingleLabel("dropTest", "help", "label")
 	_ = NewGaugesWithSingleLabel("dropTest", "help", "label")
 }
+
+func TestStringMapToString(t *testing.T) {
+	assert.Equal(t, stringMapToString(map[string]string{"foo": "bar"}), "{\"foo\": \"bar\"}")
```

Comment:
This will be flaky because map iteration does not guarantee order. You'll need to test for one of both outcomes.